nats/common.go

```diff
@@ -0,0 +1,54 @@
+package nats
+
+import "github.com/nats-io/nats.go"
+
+// Default headers which published in messages from API backend
+const (
+	RegionHdr     = "region"
```

Comment:
@sm-amykhalkiv do you need to export these constants? Maybe the constants could be replaced with the literals then?
```go
func getRegionFromHeader(m *nats.Msg) string {
	return getHeaderValue(m, "region")
}
```

Reply:
@antonsapyanov I don't right now to export them, jut to use helper functions